sim/deathknight/heart_strike.go

```diff
@@ -5,25 +5,21 @@ import (
 	"github.com/wowsims/wotlk/sim/core/stats"
 )
 
-var HeartStrikeActionID = core.ActionID{SpellID: 55050}
+var HeartStrikeActionID = core.ActionID{SpellID: 55262}
 
 func (dk *Deathknight) newHeartStrikeSpell(isMainTarget bool, isDrw bool, onhit func(sim *core.Simulation, spell *core.Spell, spellEffect *core.SpellEffect)) *RuneSpell {
-	bonusBaseDamage := 250.0 + dk.sigilOfTheDarkRiderBonus()
-	weaponBaseDamage := core.BaseDamageFuncMeleeWeapon(core.MainHand, true, bonusBaseDamage, true)
-	if !isMainTarget {
-		weaponBaseDamage = core.BaseDamageFuncMeleeWeapon(core.MainHand, true, bonusBaseDamage, true)
-	}
+	bonusBaseDamage := dk.sigilOfTheDarkRiderBonus()
+	weaponBaseDamage := core.BaseDamageFuncMeleeWeapon(core.MainHand, true, 736.0+bonusBaseDamage, true)
 
 	diseaseMulti := dk.dkDiseaseMultiplier(0.1)
 	weaponMulti := 0.5
 	if !isMainTarget {
 		weaponMulti = 0.25
 	}
 
-	outcomeApplier := dk.OutcomeFuncMeleeSpecialHitAndCrit(dk.critMultiplierGoGandMoM())
+	outcomeApplier := dk.OutcomeFuncMeleeSpecialHitAndCrit(dk.bonusCritMultiplier(dk.Talents.MightOfMograine))
 	if isDrw {
-		outcomeApplier = dk.RuneWeapon.OutcomeFuncMeleeSpecialHitAndCrit(
-			dk.RuneWeapon.MeleeCritMultiplier(1.0, dk.secondaryCritModifier(dk.Talents.GuileOfGorefiend > 0, dk.Talents.MightOfMograine > 0)))
+		outcomeApplier = dk.RuneWeapon.OutcomeFuncMeleeSpecialHitAndCrit(dk.RuneWeapon.DefaultMeleeCritMultiplier())
```

Comment:
You have removed Might of Mograine affecting DRW strikes? Are you sure this is correct

Reply:
That's a case of "since RuneWeapon was handled inconsistently, it just gets default multipliers now". I'd reason it's easier to fix once someone knows which critMultipliers actually applies to DRW, and which don't.